poetry.lock

```diff
@@ -522,7 +522,7 @@ reference = "mlcommons"
 
 [[package]]
 name = "demo-plugin"
-version = "0.2.4"
+version = "0.3.1"
```

Comment:
Double checking this was intentional just because the diff is collapsed.

Reply:
Yep, those are desirable.